src/docsfy/storage.py

```
@@ -24,15 +29,35 @@ async def init_db() -> None:
+                ai_model TEXT,
                 created_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
                 updated_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP
             )
         """)
+
+        # Migrate old databases: add columns if they don't exist
+        for column in ["ai_provider TEXT", "ai_model TEXT", "current_stage TEXT"]:
+            try:
+                await db.execute(f"ALTER TABLE projects ADD COLUMN {column}")
+            except Exception:
+                pass  # Column already exists
```

Ruff (0.15.2)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)